templates/onboarding/index.html

```diff
@@ -18,12 +18,23 @@
     <h2 class="text-dark" style="font-size:2rem">Let's take back control of your inbox!</h2>
   </div>
 
-  <!-- Button container -->
   <div class="mt-8 text-center">
-    <a class="mx-6 p-4 text-decoration-none" style="background:black;color:white;" href="{{ url_for('auth.register', next=url_for('onboarding.final')) }}">Create a new account</a>
-    <a class="mx-6 p-4 text-decoration-none" style="background:white;color:black;border-radius: 2px;border:1px solid black;" href="{{ url_for('auth.login', next=url_for('onboarding.final')) }}">I already have an account</a>
+  {% if is_user_logged_in %}
```

Comment:
Optional Nits: jinja2 has access to `current_user` so we can use it directly here.

Reply:
Changed it and removed the `is_user_logged_in` from the controller